schemas/questions/types/content/content.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This file is essentially a duplicate of `schemas/questions/types/content.json`.

Reply:
Sorry, I had meant to delete that folder